magstock/__init__.py

```diff
@@ -57,7 +65,7 @@ def food_cost(self):
 
     @property
     def auto_food(self):
-        return self.badge_type in [c.STAFF_BADGE, c.GUEST_BADGE]
+        return self.badge_type in [c.STAFF_BADGE, c.GUEST_BADGE] or hasattr(self, "auto_food_extra") and self.auto_food_extra
```

Comment:
it would be cool if the band plugin could actually override this function as if it were a derived class and then call the base function, but this was the best way I could figure out how to do it.